Module/uti/agent_belt.uti.json

```diff
@@ -2,7 +2,7 @@
   "__data_type": "UTI ",
   "AddCost": {
     "type": "dword",
-    "value": 0
+    "value": 375
```

Comment:
Agent belts are dropped on Hutlar which is a level 30-35. The pricing should be closer aligned to the T3/T4 pricing for standard items. It's currently worth over double of those items. T3 is 63cr, T4 is 157cr. A number somewhere in this range seems more applicable to me.

Reply:
"All prices have been set to 2.5x in the files to reflect the 0.4x modifier that vendors buy items at."

40% of 375 is 150, which is the correct buying price for a tier 3 loot item according to the table above that you okay'd. IE: the player will only receive 150 credits for selling this item.